plugins/modules/ec2_vpc_nat_gateway.py

```diff
@@ -204,11 +234,17 @@
 except ImportError:
     pass  # Handled by AnsibleAWSModule
 
-from ansible.module_utils._text import to_native
+
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AWSRetry
```

Comment:
Why not -

```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AWSRetry, camel_dict_to_snake_dict
```

Reply:
same of other imports.

Reply:
a) My experience has been that with imports all on one line merge conflicts are more likely
b) because camel_dict_to_snake_dict should be imported from `ansible.module_utils.common.dict_transformations` ;)

I've been working on OpenStack and AWS modules and have found that the guidelines from OpenStack make rebasing much easier.  The OpenStack folks are following https://docs.openstack.org/hacking/0.13.0/#real-world-import-order-examples